scalardb/src/scalardb/core.clj

```diff
@@ -41,25 +55,27 @@
                                                            :tx_created_at :bigint
                                                            :primary-key   [:tx_id]}))))
 
+(defn setup-transaction-tables
+  [test schemata]
+  (let [session (alia/connect
+                 (alia/cluster {:contact-points (->> test :nodes (map name))}))]
+    (doseq [schema schemata]
+      (create-transfer-table test session schema))
+    (create-coordinator-table test session)
+    (alia/shutdown session)))
+
 (defn- create-properties
   [nodes]
-  (let [props (Properties.)]
-    (if (= (count nodes) 1)
-      (.setProperty props
-                    "scalar.database.contact_points"
-                    (first nodes))
-      (.setProperty props
-                    "scalar.database.contact_points"
-                    (reduce #(str %1 "," %2) nodes)))
-    (.setProperty props "scalar.database.username" "cassandra")
-    (.setProperty props "scalar.database.password" "cassandra")
-    props))
+  (doto (Properties.)
+    (.setProperty "scalar.database.contact_points" (reduce #(str %1 "," %2) nodes))
```

Comment:
I am not sure, but I have the feeling that this may not work if `nodes` consists of only one node. It may be better to use `(clojure.string/join "," nodes)`.

Reply:
I've checked it works, but it is better to use `string/join`.
Thanks!